docs/charts/m_u_parameters_chart.ipynb

```diff
@@ -168,15 +165,48 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### What the chart shows\n"
+    "### What the chart shows\n",
+    "\n",
+    "The `m_u_parameters_chart` shows the results of a trained Splink model:\n",
+    "\n",
+    "- The left chart shows the estimated m probabilities from the Splink model \n",
+    "- The right chart shows the estimated u probabilities from the Splink model.\n",
+    "\n",
+    "Each comparison within a model is represented in trained m and u values that have been estimated during the Splink model training for each comparison level.\n",
+    "\n",
+    "??? note \"What the chart tooltip shows\"\n",
+    "\n",
+    "    ![](./img/m_u_parameters_chart_tooltip_1.png)\n",
+    "\n",
+    "    The tooltip of the left chart shows information based on the comparison level bar that the user is hovering over, including:\n",
+    "\n",
+    "    - An explanation of the m probability for the comparison level.\n",
+    "    - The name of the comparison and comparison level.\n",
+    "    - The comparison level condition as an SQL statement.\n",
+    "    - The m and u proability for the comparison level.\n",
+    "    - The resulting bayes factor and match weight for the comparison level.\n",
+    "\n",
+    "    ![](./img/m_u_parameters_chart_tooltip_2.png)\n",
+    "\n",
+    "    The tooltip of the right chart shows information based on the comparison level bar that the user is hovering over, including:\n",
+    "\n",
+    "    - An explanation of the u probability from the comparison level.\n",
+    "    - The name of the comparison and comparison level.\n",
+    "    - The comparison level condition as an SQL statement.\n",
+    "    - The m and u proability for the comparison level.\n",
+    "    - The resulting bayes factor and match weight for the comparison level."
```

Comment:
Should we label these with subtitles? At the moment it's a tad confusing working out where each image has come from at a glance (this is clarified below, but it's not as obvious when scanning).

I would consider adding some simplistic titles such as:
`Estimated m probability tooltips` and `Estimated u probability tooltips`. 

These suggestions differ from the subtitles already in place on the chart, so you may want to mirror those instead for clarity.

Reply:
Maybe expand on the m/u descriptions? 

Some additional commentary on precisely what the %s mean in practice would be useful for users.

Links to other areas of the docs discussing this would also work.

Reply:
On https://github.com/moj-analytical-services/splink/pull/1551/files#r1317430088, if you feel the description below on "How to interpret the chart" is sufficient, you can ignore this.

Reply:
Have added a link to the fellegi sunter docs